api/tests/unit/util/mappers/test_unit_mappers_sdk.py

```diff
@@ -0,0 +1,115 @@
+from typing import TYPE_CHECKING
+
+import pytest
+
+from environments.identities.models import Identity
+from util.mappers.sdk import map_environment_to_sdk_document
+
+if TYPE_CHECKING:  # pragma: no cover
+    from pytest_mock import MockerFixture
+
+    from environments.models import Environment
+    from features.models import FeatureState
+
+
+@pytest.fixture()
+def identity_without_overrides(environment):
+    return Identity.objects.create(
+        identifier="test_identity_without_overrides", environment=environment
+    )
+
+
+def test_map_environment_to_sdk_document__return_expected(
+    mocker: "MockerFixture",
+    environment: "Environment",
+    feature_state: "FeatureState",
+    identity: Identity,
+    identity_featurestate: "FeatureState",
+    identity_without_overrides: Identity,
+) -> None:
+    # Given
+    expected_overridden_value = "some-overridden-value"
+    expected_api_key = environment.api_key
+
+    identity_featurestate.feature_state_value.string_value = expected_overridden_value
+    identity_featurestate.feature_state_value.save()
+    identity_featurestate.enabled = True
+    identity_featurestate.save()
+
+    identities_with_overrides = (
+        Identity.objects.only_overrides().filter(environment=environment).all()
+    )
+
+    # When
+    result = map_environment_to_sdk_document(
+        environment,
+        identities_with_overrides=identities_with_overrides,
+    )
```

Comment:
I'd love to see an assertion on the number of queries here (along with maybe multiple identities / identity overrides).

Reply:
This is a valid request but the assertion won't belong to the mapper test as it expects already fetched data with the `identities_with_overrides` kwarg. I'll add multiple overrides to the environment-document view test.

Reply:
Done, and fixed an N+1 issue in the process 😬